core/backend/src/CheckpointManager.ts

```diff
@@ -209,14 +209,19 @@ export class V2CheckpointManager {
         container.connect(this.cloudCache);
       container.checkForChanges();
       if (IModelHost.appWorkspace.settings.getBoolean("Checkpoints/prefetch", false)) {
-        const logPrefetch = async (prefetch: CloudSqlite.CloudPrefetch) => {
+        // eslint-disable-next-line @typescript-eslint/no-shadow
```

Comment:
I was getting a no-shadow lint error. I'm assuming its safe to disable that here? I don't see how the outer scope would affect the inner scope of minRequests, maxRequests, timeout.

Reply:
Thanks